### PR TITLE
[FIX] web: prevent scrollbars for kanban ghost

### PR DESCRIPTION
**Pull request by kbapt**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by tivisse** [comment]
@qsm-odoo Could you please take a look at this small fix ? 

Thanks.

**Comment by qsm-odoo** [comment]
@aab-odoo Sorry to ping you but you probably know more about the structure of what is designed-fix here (as you made this https://github.com/odoo/odoo/commit/dd7022eccd99b5ba9688f63b517d23c405f23797).

**Comment by aab-odoo** [comment]
@kba-odoo i think that the issue comes from hr_recruitment... [This rule](https://github.com/odoo/odoo/blob/13.0/addons/hr_recruitment/static/src/scss/hr_job.scss#L5) should not apply on .o_kanban_ghost

**Comment by kba-odoo** [comment]
@aab-odoo indeed you are correct. I've made the necessary changes now for hr_recruitment

**Comment by tivisse** [comment]
@robodoo r+

**Comment by aab-odoo** [comment]
robodoo r-

File: addons/web/static/src/scss/search_panel.scss [hunk 1]
```diff
@@ -15,6 +15,7 @@ $o-searchpanel-filter-default-color: #D59244;
         flex: 1 1 100%;
         overflow: auto; // make the renderer and search panel scroll individually
         max-height: 100%;
+        height: 100%;
```
Comment (aab-odoo) [inline addons/web/static/src/scss/search_panel.scss]: do you need this?

Comment (kba-odoo) [reply addons/web/static/src/scss/search_panel.scss]: It's for the search panel to take the full height:
 - Without: https://drive.google.com/open?id=1Av4ivMuoNbLLXlUPZFVsRc2HdooA7j8T
 - With: https://drive.google.com/open?id=18Fr-d7ItSmBcvHZsx1Kkv1GmvsgbFpgq

Comment (aab-odoo) [reply addons/web/static/src/scss/search_panel.scss]: I can't reproduce on the last v13 runbot...

Comment (kba-odoo) [reply addons/web/static/src/scss/search_panel.scss]: Sorry I mixed multiple things... It was rather to fix this issue: https://drive.google.com/open?id=1q0X7qKpTqeO_XC4NhiuES6z2vrs2WMcu but it seems it's no longer an issue on the latest runbot!

My bad, I'll remove the height: 100% here

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**